qa/suites/fs/functional/tasks/admin.yaml

```diff
@@ -10,6 +10,7 @@ overrides:
       - \(MDS_CLIENTS_BROKEN_ROOTSQUASH\)
       - report clients with broken root_squash implementation
       - evicting unresponsive client
+      - as file system flag refuse_client_session is set
```

Comment:
https://pulpito.ceph.com/jcollin-2024-07-12_00:07:25-fs-wip-jcollin-testing-20240711.095637-squid-distro-default-smithi/7798072/

This this again, I'm thinking instead of just ignore this warning shouldn't we just fix the test cases ? 

For example:

```
1015     def test_fs_rename_fails_for_non_existent_fs(self):
1016         """
1017         That renaming a non-existent file system fails.                                                                                                                                                                                                  
1018         """
1019         self.run_ceph_cmd(f'fs fail {self.fs.name}')
1020         self.run_ceph_cmd(f'fs set {self.fs.name} refuse_client_session true')
1021         sleep(5)
1022   
1023         try:
1024             self.run_ceph_cmd("fs rename non_existent_fs new_fs --yes-i-really-mean-it")
1025         except CommandFailedError as ce:    
1026             self.assertEqual(ce.exitstatus, errno.ENOENT, "invalid error code on renaming a non-existent fs")
1027         else:
1028             self.fail("expected renaming of a non-existent file system to fail")
1029         self.run_ceph_cmd(f'fs set {self.fs.name} joinable true')
1030         self.fs.wait_for_daemons()
1031         self.run_ceph_cmd(f'fs set {self.fs.name} refuse_client_session false')
```
Since the mountpoints won't used in the test cases, we should just unmount them in the beginning ?

Reply:
The clients are supposed to reconnect back (and the MDS should allow that) once the flag is unset, so unmounting clients is kind of taking away that check.